### PR TITLE
Add xhosa translations

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         bot: ['base']
-        lang: ['eng', 'afr']
+        lang: ['eng', 'afr', 'xho']
```
Comment (erikh360) [inline .github/workflows/build.yaml:14]: We probably don't need this on the build step as all the models are the same. We can just reference the english model in all the workers, thoughts?

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         bot: ['base']
-        lang: ['eng', 'afr']
+        lang: ['eng', 'afr', 'xho']
```
Comment (erikh360) [inline .github/workflows/test.yaml:11]: not even sure if this is really needed

File: .github/workflows/build.yaml (hunk 1 above)
Comment (rudigiesler) [reply .github/workflows/build.yaml:14]: No, the domain file is included in the model, so we have to build a model for each language.

File: .github/workflows/test.yaml (hunk 2 above)
Comment (rudigiesler) [reply .github/workflows/test.yaml:11]: Possibly not, but we do a `rasa data validate` on the domain file, so probably good to have that for each domain file to validate that it's correct

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**